rspack.config.js

```diff
@@ -67,6 +67,12 @@ const CSS_CONFIG = {
   modules: {
     auto: filename =>
       !filename.includes("node_modules") && !filename.includes("vendor.css"),
+    mode: filename => {
```

Comment:
This allows us to write `.global.css` files that are css module files where all classes are global by default.

This helps avoid having to wrap each class in a `:global` wrapper (as would be the case in `AceEditor.global.css`.